input/drivers/sdl_input.c

```diff
@@ -549,13 +729,100 @@ static uint64_t sdl_get_capabilities(void *data)
          | (1 << RETRO_DEVICE_ANALOG);
 }
 
+static bool sdl_input_set_sensor_state (void *data, unsigned port, enum retro_sensor_action action, unsigned rate) {
+   return true;
+}
```

Comment:
Can you prefix this type name with `sdl_input_`? Doing so will mitigate the risk of clashing names in griffin builds (which effectively `#include` all of RetroArch into one `.c` file, simplifying the build for certain platforms).